test_runner/regress/test_aux_files.py

```diff
@@ -0,0 +1,81 @@
+from fixtures.log_helper import log
+from fixtures.neon_fixtures import (
+    NeonEnv,
+    logical_replication_sync,
+)
+
+
+def test_aux_v2_config_switch(neon_simple_env: NeonEnv, vanilla_pg):
+    env = neon_simple_env
+
+    tenant_id = env.initial_tenant
+    timeline_id = env.neon_cli.create_branch("test_aux_v2_config_switch", "empty")
```

Comment:
Does the neon_simple_env create the "empty" branch? With neon_env_builder it's always "main".

But anyways, there is no need to create a branch, there is one already.